orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/aws/lambda/lambdaFunctionTask.java

```diff
@@ -44,8 +51,14 @@ public TaskResult execute(@Nonnull Stage stage) {
 
     Map<String, Object> task = new HashMap<>(stage.getContext());
 
+    String operationName = (String) task.get("operation");
+    if (!StringUtils.isNotBlank(operationName)) {
```

Comment:
The double negative is a bit confusing, `if (StringUtils.isBlank(operationName))` is > readable